decidim-meetings/app/helpers/decidim/meetings/directory/application_helper.rb

```diff
@@ -92,7 +92,7 @@ def directory_filter_categories_values
             key_point = current_participatory_space.class.name.gsub("::", "__") + current_participatory_space.id.to_s
 
             TreeNode.new(
-              TreePoint.new(key_point, translated_attribute(current_participatory_space.title, current_organization)),
+              TreePoint.new(key_point, decidim_escape_translated(current_participatory_space.title)),
```

Comment:
Seems like this change isn't in the original PR and neither in develop, so we should implement it there first just to be sure

https://github.com/decidim/decidim/blob/a068bbb39f70d5b56c6ab0c4d4c0618c71d4270f/decidim-meetings/app/helpers/decidim/meetings/directory/application_helper.rb#L62

Reply:
Actually, this has been implemented by the redesign team, and this is why we never caught it before. 

The method called `directory_filter_categories_values` defined in application_helper is being used with the following stackTrace: 
1. decidim-meetings/app/views/decidim/meetings/shared/_filters.html.erb:7
2. decidim-core/app/views/decidim/shared/_filters.html.erb:35
3. decidim-core/app/views/decidim/shared/filters/_check_boxes_tree.html.erb:10
4. decidim-core/lib/decidim/filter_form_builder.rb:49
5. decidim-core/app/views/decidim/shared/filters/_dropdown_label.html.erb:23
6. decidim-core/app/views/decidim/shared/filters/_dropdown_label.html.erb:6

the method ` filter_text_for(translation, id: nil)`, is rendering a content_tag with a translation parameter that is automatically escaped by ERB. 
in 0.27, we do not have ` filter_text_for` therefore we need to handle by hand ( which i did here ) 

Looking at the tree helper, there is a test improvement that can be done for main elements + children so that we have everything escaped properly. This would be another PR against the develop, that needs to happen at some point, and i would not keep this PR any longer.

Reply:
>  This would be another PR against the develop, that needs to happen at some point, and i would not keep this PR any longer.

I agree. Can you create an issue with this so we don't have any pending conversation 🙏🏽 ? You have a much better knowledge of what's happening here and what should be the final result than me to be honest 😃

Reply:
@andreslucena  Done in #12674

Reply:
Thanks!